rabbitmq/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # CHANGELOG - rabbitmq
 
 
+1.3.1 / Unreleased
+==================
+
+### Changes
+
+* [BUGFIX] Add a key check before updating connection state metric. See [#729][]
```

Comment:
You also need to add a link to #729 at the end of the file.

Reply:
Done. Was under the impression that a certain PimpMyChangelog does it automagically